lib/product.rb

```diff
@@ -0,0 +1,20 @@
+class Product
+  attr_reader :category, :name, :unit_price, :quantity, :hoard
+  def initialize(category, name, unit_price, quantity)
+    @category = category
+    @name = name
+    @unit_price = unit_price
+    @quantity=  quantity.to_i
+    hoard = true
```

Comment:
When we create this it should be an instance variable. So we need to use the `@hoard` syntax. Also based off of the interaction patter in the read me a product `@is_hoarded` should be false be default.